demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1625,14 +1594,15 @@ static void prvDisconnectFromMQTTBroker( void )
     /* End TLS session, then close TCP connection. */
     ( void ) SecureSocketsTransport_Disconnect( &networkContextMqtt );
```

Comment:
The "networkContextMqtt" variable should be "xNetworkContextMqtt"

Reply:
Thanks, updated in https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724